StockAnalyzerService/StockAnalyzerService/Service/StockAnalyzer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using StockAnalyzerService.Model;
+using StockAnalyzerService.Service;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace StockAnalyzerService.Service {
+	public class StockAnalyzer : IStockAnalyzer {
+		public IHttpClientFactory clientFactory;
+		public IHttpCalls httpCalls;
+		public StockAnalyzer(IHttpClientFactory _clientFactory, IHttpCalls _httpCalls) {
+			clientFactory = _clientFactory;
+			httpCalls = _httpCalls;
+		}
+
+		public async Task<List<StockMetadata>> GetStocksWithUsers() {
```

Comment:
Does this method name fit the purpose of the class, StockAnalyzer?

Reply:
It will be used later during actual analysis methods

Reply:
Right, but the class is met for analysis, not having methods to retrieve data

Reply:
It would end up meaning more stuff to put in the constructor and then editing all the tests for one or two methods. I feel like getting the data is required for actually analyzing it even if you aren't doing any analysis in that method